cli/cmd/start.go

```diff
@@ -68,7 +68,7 @@ func internalStartModule(cmdCtx *cmdcontext.CmdCtx, args []string) error {
 	}
 
 	log.Info("Starting an instance...")
-	if err = running.Start(cmdCtx); err != nil {
+	if err = running.Start(cmdCtx, args); err != nil {
```

Comment:
Why do you need to pass the `args` to the start?

Reply:
Changed